src/shared/components/data-table/DataTable.scss

```diff
@@ -0,0 +1,54 @@
+@import 'src/styles/common';
+
+.wrapper {
+  overflow: auto;
+
+  &ThemeDark {
+    background-color: $light-grey;
+  }
+}
+
+.tableContainer {
+  display: block;
```

Comment:
That's default value for a div

Reply:
Removed it.

Also, I've fixed the sequence alignment margin.